NewHorizons/Builder/Atmosphere/AirBuilder.cs

```diff
@@ -57,6 +57,7 @@ public static void Make(GameObject planetGO, Sector sector, PlanetConfig config)
                 airGO.AddComponent<OxygenVolume>()._treeVolume = config.Atmosphere.hasTrees;
             }
 
+            // TODO: figure out what to change "config.Atmosphere.hasRain" to...
```

Comment:
im assuming this pr is blocked on this?

Reply:
yes

Reply:
I'm not sure whether I should keep it or just make it obsolete and people will have to figure out how to add the vection field, rain audio, and visor effects themselves.

Reply:
Just leave hasRain as is